### DISCUSSION TITLE
Image on map

### DISCUSSION THREAD

**Discussion by PopescuGeorge94** [Mapsui/Mapsui#2160] [Q&A]
You can add an image (PNG or JPEG) over the map?

**Comment by inforithmics** [comment]
https://mapsui.com/samples/

Data Formats
Single Image

Yes you can

**Comment by PopescuGeorge94** [comment]
Sorry, I didn't express myself correctly, the image should be a static image in the corner of the map.

**Comment by PopescuGeorge94** [comment]
I resolved the "problem"

Here is the code

            var buttonWidget = new ButtonWidget
            {
                HorizontalAlignment = Mapsui.Widgets.HorizontalAlignment.Right,
                VerticalAlignment = Mapsui.Widgets.VerticalAlignment.Top,
                MarginX = 10,
                MarginY = 20,
                Width = 80,
                Height = 50,
                Opacity = 1,
                Text = "Locate Me",
                Picture = EmbeddedResourceLoader.Load("Images.MyLocationStill.svg", typeof(Program)).LoadSvgPicture(),
                TextColor = new Color(65, 105, 225)
            };

            mapControl.Map.Widgets.Add(buttonWidget);
